pkg/test/masteruserrecord/assertion.go

```diff
@@ -194,3 +200,21 @@ func (a *Assertion) DoesNotHaveFinalizer() *Assertion {
 	assert.Len(a.t, a.masterUserRecord.Finalizers, 0)
 	return a
 }
+
+// DoesNotHaveLabel verifies that the MasterUserRecord does not have
+// a label with the given key
+func (a *Assertion) DoesNotHaveLabel(key string) *Assertion {
+	err := a.loadUaAssertion()
+	require.NoError(a.t, err)
+	assert.NotContains(a.t, a.masterUserRecord.Labels, key)
+	return a
+}
+
+// HasLabel verifies that the MasterUserRecord has
+// a label with the given key
+func (a *Assertion) HasLabel(key string) *Assertion {
```

Comment:
it would be also good to have an assertion:
```go
 func (a *Assertion) HasLabelWithValue(key, value string) *Assertion {
```
that would verify also the value. This could be used here https://github.com/codeready-toolchain/host-operator/pull/221/files#diff-dd63ae5743a88e29925009afded0e819R85 to verify that the label still exists with the same value as was before

Reply:
never mind, I've just realized that the value is verified as part of `UserAccountHasTier`

Reply:
ok, marking as resolved for now then